sdks/wasm/Makefile

```diff
@@ -168,7 +174,7 @@ build-debug-sample: .stamp-build-debug-sample
 
 build-test-suite: .stamp-build-test-suite
 
-build-managed: build-debug-sample build-test-suite
+build-managed: build-debug-sample build-test-suite $(WASM_BCL_DIR)/monolinker.exe
```

Comment:
Addressed by Zoltan's latest commit

Reply:
I am not sure this is much better. What mono runtime (from where) are you going to use to run it?

Reply:
That's just wrong, you cannot bundle version which is built with master BCL it might not run on anything else than that specific version. If you tell me what the real intentions are then I can suggest how to do it properly as we have different setups for this based on the use case.

Reply:
We will probly mkbundle it.

Reply:
Why (it will just cause more troubles later)? Just do what other xamarin tools do

Reply:
We'll probably keep the usage of CLI tools as a mono hacking only thing.
wasm will ship with msbuild targets and tasks that will consume the linker as a library and not as a standard alone process.

If we're making consideration for customer usage, we must keep in mind that this is will be primarily used on windows.